.github/workflows/development-workflow.yml

```diff
@@ -0,0 +1,129 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: "Development Workflow"
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+  release:
+    types: [released]
+
+defaults:
+  run:
+    working-directory: "./"
+jobs:
+  dependencies:
+    runs-on: ubuntu-latest
+    name: Download dependencies
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            **/node_modules
+          key: ${{github.sha}}-dependencies
+      - run: |
+          npm ci
+          npm run bootstrap
+  lint:
+    runs-on: ubuntu-latest
+    needs: [dependencies]
+    name: Linting
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            **/node_modules
+          key: ${{github.sha}}-dependencies
+      - run: |
+          npm run lint
+  # test:
+  #   runs-on: ubuntu-latest
+  #   needs: [dependencies]
+  #   name: Testing
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: actions/setup-node@v1
+  #       with:
+  #         node-version: 12.x
+  #     - uses: actions/cache@v2
+  #       with:
+  #         path: |
+  #           **/node_modules
+  #         key: ${{github.sha}}-dependencies
+  #     - run: npm run test-ci
```

Comment:
Can we remove the comments here, and maybe add them later

(Applicable for all comments)

Reply:
Well we this is not something which we don't want and for that reason we are comment, it is not working currently because of an issue with the tests. So, once we resolve the issue we should enable the test job.